src/webgpu/api/validation/createBindGroup.spec.ts

```diff
@@ -266,15 +267,22 @@ g.test('texture_must_have_correct_dimension')
       ],
     });
 
+    let height = 16;
+    let depthOrArrayLayers = 6;
+    if (dimension === '1d') {
+      height = 1;
+      depthOrArrayLayers = 1;
+    }
+
     const texture = t.device.createTexture({
-      size: { width: 16, height: 16, depthOrArrayLayers: 6 },
+      size: { width: 16, height, depthOrArrayLayers },
       format: 'rgba8unorm' as const,
       usage: GPUTextureUsage.SAMPLED,
+      dimension: getTextureDimensionFromView(dimension),
     });
 
     const shouldError = viewDimension !== dimension;
-    const arrayLayerCount = dimension === '2d' ? 1 : undefined;
```

Comment:
No need to set arrayLayerCount, WebGPU spec has rules to set viewDesc.arrayLayerCount if it is undefined.